GitCommands/Settings/AppSettings.cs

```diff
@@ -2104,9 +2104,12 @@ private static void ImportFromRegistry()
         [Obsolete("AppSettings is no longer responsible for colors, ThemeModule is")]
         public static Color GetColor(AppColor name)
         {
-            return SettingsContainer.GetColor(name.ToString().ToLowerInvariant() + "color", AppColorDefaults.GetBy(name));
+            return SettingsContainer.GetColor(GetColorSettingName(name), AppColorDefaults.GetBy(name));
         }
 
+        [Obsolete("AppSettings is no longer responsible for colors, ThemeModule is")]
+        public static string GetColorSettingName(AppColor color) => color.ToString().ToLowerInvariant() + "color";
```

Comment:
Please do not publicly expose methods just for tests.
If
```suggestion
        internal static string GetColorSettingName(AppColor color) => color.ToString().ToLowerInvariant() + "color";
```
does not work, the TestAccessor pattern should be applied.
But in this case, just duplicate the regarding code in the test.

Reply:
I just removed it. The name of the setting doesn't matter really, just that it doesn't exist.